apisix/plugins/log-rotate.lua

```diff
@@ -189,10 +189,8 @@ end
 local function init_default_logs(logs_info, log_type)
     local filepath, filename = get_log_path_info(log_type)
     logs_info[log_type] = { type = log_type }
-    if filename ~= "off" then
```

Comment:
why did you remove this line?

Reply:
Fixed, I remove it for test and forget to restore it.

Reply:
@shreemaan-abhishek Could you please review this PR? Thank you!